rudder-agent/debian/rules

```diff
@@ -26,18 +26,20 @@ configure-stamp: get-flags
 	# If there is no configure, bootstrap with autogen.sh first
 	cd SOURCES/cfengine-source && [ -x ./configure ] || NO_CONFIGURE=1 ./autogen.sh
 	# Prepare CFEngine 3.6.x build with defined path of LMDB
-	cd SOURCES/cfengine-source && ./configure --prefix=/opt/rudder --with-workdir=/var/rudder/cfengine-community --enable-static=yes --enable-shared=no --with-lmdb=$(CURDIR)/debian/tmp/opt/rudder CFLAGS="$(SECURE_CFLAGS)" LDFLAGS="$(SECURE_LDFLAGS)"
+	cd SOURCES/cfengine-source && ./configure --prefix=/opt/rudder --with-workdir=/var/rudder/cfengine-community --enable-static=yes --enable-shared=no --with-lmdb=$(CURDIR)/debian/tmp/opt/rudder CFLAGS="$(SECURE_CFLAGS) $(BUILD_CFLAGS)" LDFLAGS="$(SECURE_LDFLAGS) $(BUILD_LDFLAGS)"
```

Comment:
Do not forget to adjust this too when you have changed the SPECfile :) (and reverse the LDFLAGS too)